Security/TwoFactor/CsrfProtection/CsrfTokenValidator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Scheb\TwoFactorBundle\Security\TwoFactor\CsrfProtection;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
+
+class CsrfTokenValidator
+{
+    /**
+     * @var string
+     */
+    private $firewallName;
+
+    /**
+     * @var array
+     */
+    private $options;
+
+    public function __construct(string $firewallName, array $options)
+    {
+        $this->firewallName = $firewallName;
+        $this->options = $options;
+    }
+
+    public function supports(string $firewallName): bool
```

Comment:
I don't see this method being used. Can be removed, right? Then we could also remove `firewallName` as an argument.

Reply:
I initially thought we needed to double check the firewall name for multi-firewall support, but since we already have a listener per firewall this shouldn't be an issue.